File: project/56/rectangle.py
import tkinter as tk

root = tk.Tk()
canvas = tk.Canvas(root, width=300, height=200)
canvas.pack()

# Creating a rectangle
rectangle = canvas.create_rectangle(50, 50, 200, 150, outline='blue', fill='red', width=2)

root.mainloop()

File: P2 - Core JAVA/java practicals/5th prac/Rectangle.java
class Area {

    int width, height;

    Area(int w, int h) {
        width = w;
        height = h;
    }

    public void showArea() {
        int A;
        System.out.println("Height : " + height + " and width is " + width);
        A = width * height;
        System.out.println("\nArea is " + A);
    }
}

class Rectangle extends Area {
    Rectangle(int w, int h) {
        super(w, h);
    }

    public static void main(String args[]) {
        Rectangle R = new Rectangle(10, 20);
        R.showArea();
    }
}